lib/test/unit/ui/console/testrunner.rb

```diff
@@ -59,6 +61,8 @@ def initialize(suite, options={})
             @faults = []
             @code_snippet_fetcher = CodeSnippetFetcher.new
             @test_suites = []
+            @report_slow_tests = @options[:report_slow_tests]
+            @report_slow_tests = false if @report_slow_tests.nil?
```

Comment:
We can remove this because `nil` is a falsy value.

Reply:
Thanks! Certainly it is needless initialize.
I've implemented in 558144f. Could you review it again?